keras_retinanet/bin/train.py

```diff
@@ -383,7 +407,8 @@ def csv_list(string):
     csv_parser.add_argument('annotations', help='Path to CSV file containing annotations for training.')
     csv_parser.add_argument('classes', help='Path to a CSV file containing class label mapping.')
     csv_parser.add_argument('--val-annotations', help='Path to CSV file containing annotations for validation (optional).')
-
+    csv_parser.add_argument('--dataset_dir', help='path to the dataset', default=None)
```

Comment:
Please use `--dataset-dir`, it will be converted to an underscore automatically.
Also, please make the help a sentence (start with a capital and end with a period), to keep it in the same style as the rest.